metron-platform/metron-data-management/README.md

```diff
@@ -250,3 +250,18 @@ The parameters for the utility are as follows:
 | -l         | --log4j             | No           | The log4j properties file to load                                                                                                                                                    |
 | -n         | --enrichment_config | No           | The JSON document describing the enrichments to configure.  Unlike other loaders, this is run first if specified.                                                                    |
 
+### GeoLite2 Loader
+
+The shell script `$METRON_HOME/bin/geo_enrichment_load.sh` will retrieve MaxMind GeoLite2 data and load data into HDFS, and update the configuration.
+
+THIS SCRIPT WILL NOT UPDATE AMBARI'S GLOBAL.JSON, JUST THE ZK CONFIGS.  CHANGES WILL GO INTO EFFECT, BUT WILL NOT PERSIST PAST AN AMBARI RESTART UNTIL UPDATED THERE.
+
```

Comment:
This gets into the whole "How do we manage configs discussion?".  Unfortunately, it's in a really awkward spot.  I might be able to add a service action to sorta take care of it, but it still probably does the same end around of Ambari's management, except it's going through the UI. I don't know that there's a good solution to this until we unify our config management, which is the real answer here.

Given that I don't think there was originally support for updating the db, I'm inclined to clean it up as part of unifying config management.  It's ugly and I don't like it, but I can't come up with a good, short-term, alternative that isn't ugly for other reason anyway.